index.html

```diff
@@ -174,95 +180,122 @@
-          match</em> and discard <em>B</em> from further matching.
+          <li>Run the <a>URL Serializer</a> with <var>urlB</var> as input and
+          <a>validate</a> the output. If validation throws, return false.
           </li>
-          <li>The identifiers match if <em>urlA</em> <strong>equals</strong>
-          <em>urlB</em> using the <a>URL equivalence</a> test (i.e. the test
-          returns true).
+          <li>Return <var>urlA</var> <a>equals</a> <var>urlB</var>.
           </li>
-        </ul>
+        </ol>
       </section>
     </section>
     <section>
       <h2>
-        W3C-Defined Strings as Payment Method Identifiers
+        Standardized Payment Method Identifiers
       </h2>
       <p>
-        W3C expects to publish a small number of payment method specifications
-        that define abstractions across similar payment methods (such as credit
-        card payments, tokenized payments, or credit transfers). W3C expects
-        these payment methods will be implemented in a wide variety of third
-        party payment apps. W3C identifies these payment methods with (short)
-        strings; W3C expects to mint these strings in payment method
-        specifications published by the Web Payments Working Group.
+        The <a href="https://w3.org">W3C</a> expects to publish a small number
+        of payment method specifications that define abstractions across
+        similar payment methods (such as credit card payments, tokenized
+        payments, or credit transfers). The <a href="https://w3.org">W3C</a>
+        expects these payment methods will be implemented in a wide variety of
+        third party payment handlers. The W3C identifies these payment methods
+        with (short) strings (individually referred to as a <dfn data-lt=
+        "standardized payment method identifiers">standardized payment method
+        identifier</dfn>); the W3C expects to mint these strings in payment
+        method specifications published by the <a href=
+        "https://www.w3.org/Payments/WG/">Web Payments Working Group</a>.
       </p>
       <section>
         <h2>
           Syntax
         </h2>
         <p>
-          The syntax of W3C strings used as payment method identifiers is
-          constrained by this regular expression:
+          The syntax of <a>standardized payment method identifier</a> used as
+          payment method identifiers is constrained by this regular expression:
         </p>
-        <pre>
-          [a-z0-9-]+
-        </pre>
+        <pre>[a-z0-9-]+</pre>
       </section>
       <section>
         <h2>
-          Matching
+          Comparison
         </h2>
         <p>
-          For W3C strings, user agents MUST use exact string matching.
+          For a<a>standardized payment method identifiers</a>, user agents MUST
+          compare strings in a <a data-cite=
+          "HTML#case-sensitive">case-sensitive</a> manner (code point for code
```

Comment:
/cc @annevk we should probably move this to Infra (no action item for now on this document though)